src/Schedulers/news.scheduler.ts

```diff
@@ -10,8 +10,22 @@ export const updateNews = async () => {
         const newsMFI = await newsScraperMFI();
 
         await NewsModel.deleteMany({});
-        await NewsModel.insertMany(newsINF);
-        await NewsModel.insertMany(newsMFI);
+        if (newsINF && newsINF.length > 0) {
```

Comment:
It's fine, but maybe better idea is to use isEmpty from Lodash, because it gives an information at first glance which case are You checking.

Reply:
When I use isEmpty from Lodash, typescript tells me that variable might be null